features/bevragen/persoon/verblijfplaats/adres/onbekendwaardes.feature

```diff
@@ -0,0 +1,108 @@
+# language: nl
+
+@proxy @ready @post-assert
+Functionaliteit: Als gebruiker van de API wil ik geen onbekend waardes ontvangen binnen het adres
```

Comment:
Is dit een goede titel? Voorstel: Niet leveren van de standaard/onbekend waarde van een adres veld

Reply:
Aangepast